www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -117,8 +131,21 @@ function (declare, Deferred, ObjectStore, Store, QueryResults, topic,
               if (reportDataList instanceof RequestFailed)
                 deferred.reject(
                   'Failed to get reports: ' + reportDataList.message);
-              else
+              else {
                 deferred.resolve(that._formatItems(reportDataList));
+
+                if (query.reportFilters.length > 1)
```

Comment:
You use this code part in three different places:
- https://github.com/whisperity/CodeChecker/blob/23639a20fdc5eef364a83dee1a162ae0ec8c841b/www/scripts/codecheckerviewer/ListOfBugs.js#L137-L147
- https://github.com/whisperity/CodeChecker/blob/23639a20fdc5eef364a83dee1a162ae0ec8c841b/www/scripts/codecheckerviewer/ListOfBugs.js#L167-L177
- https://github.com/whisperity/CodeChecker/blob/23639a20fdc5eef364a83dee1a162ae0ec8c841b/www/scripts/codecheckerviewer/ListOfBugs.js#L197-L207

I think you should create a function for this.